app/main.py

```diff
@@ -1 +1,60 @@
-# write your code here
+class SoftwareEngineer:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.name = name
```

Comment:
Redundant assignment: `self.name = name` is already handled by the `super().__init__(name)` call in line 14. You can remove this line to avoid redundancy.